src/Components/WithCurrentTime.tsx

```diff
@@ -36,15 +51,30 @@ export class WithCurrentTime extends React.Component<
     )
   }
 
+  async componentWillMount() {
```

Comment:
This should be `componentDidMount` so it doesn't run server-side.

Reply:
Oh good call! Was just reading about how did mount is better than will mount for these sorts of 'initial fetches'.